foodsaving/groups/factories.py

```diff
@@ -14,7 +14,7 @@ def members(self, created, members, **kwargs):
             return
         if members:
             for member in members:
-                self.members.add(member)
+                GroupMembership.objects.create(group=self, user=member)
```

Comment:
This occurs quite a lot. Maybe use the `add_member(user)` helper method to the Group model and modify it to send out the signal only if a parameter is set?

Reply:
I thought about it, but didn't want to open up the topic at this point.

At the moment our signal use is inconsistent as I've used all the built in django ones, but the existing code had defined custom ones. I didn't think about it much lately.

https://stackoverflow.com/a/26490827 has a way to disable signals during factory setup.

Maybe using the add_member helper and suppressing the signals would be better at the moment.

Reply:
I pulled it out into it's own issue, because the solution was not clear to me https://github.com/yunity/foodsaving-backend/issues/361

So, if you're happy I would propose to merge this as it is now, and rework it depending on the outcome of #361?